### PR TITLE
remove references to Jupyter notebooks within the Druid repo

### PR DESCRIPTION
**Pull request by techdocsmith**
This PR removes the links to the Jupyter notebooks within the Druid repo and points to the Learn Druid repo.

cc: @sergioferragut 

This PR has:

- [x ] been self-reviewed.

### CODE REVIEW

File: docs/tutorials/tutorial-sql-query-view.md [hunk 1]
```diff
@@ -193,3 +193,5 @@ For more information on ingestion and querying data, see the following topics:
 - [Ingestion](../ingestion/index.md) for an overview of ingestion and the ingestion methods available in Druid.
 - [SQL-based ingestion](../multi-stage-query/index.md) for an overview of SQL-based ingestion.
 - [SQL-based ingestion query examples](../multi-stage-query/examples.md) for examples of SQL-based ingestion for various use cases.
+- Try out the interactive [Introduction to Druid SQL](https://github.com/implydata/learn-druid/tree/main/notebooks) notebook to learn more about Druid SQL.
```
Comment (ektravel) [inline docs/tutorials/tutorial-sql-query-view.md:196]: Line 196 doesn't follow the same format as the rest of the "Further reading" entries. Consider changing it to the following:

- [Introduction to Druid SQL](https://github.com/implydata/learn-druid/tree/main/notebooks) to learn more about Druid SQL.

File: docs/tutorials/tutorial-jupyter-index.md [hunk 2]
```diff
@@ -23,53 +23,10 @@ sidebar_label: Jupyter Notebook tutorials
   ~ under the License.
   -->
 
-<!-- tutorial-jupyter-index.md and examples/quickstart/juptyer-notebooks/README.md
-    share a lot of the same content. If you make a change in one place, update the other
-    too. -->
+You can try out the Druid APIs using interactive Jupyter Notebook tutorials.
```
Comment (ektravel) [inline docs/tutorials/tutorial-jupyter-index.md:26]:
```suggestion
You can try out the Druid APIs using interactive Jupyter notebook tutorials.
```

I don't think we capitalize "notebook" anywhere else in the docs.

File: docs/tutorials/tutorial-jupyter-index.md [hunk 3]
```diff
@@ -23,53 +23,10 @@ sidebar_label: Jupyter Notebook tutorials
...
+These tutorials provide snippets of Python code that you can use to run calls against the Druid API to complete the tutorial.
```
Comment (ektravel) [inline docs/tutorials/tutorial-jupyter-index.md:27]:
```suggestion
These tutorials provide snippets of Python code that you can use to run calls against the Druid API.
```
"to complete the tutorial" is unnecessary

**Review by ektravel — COMMENTED** [review]
Left some suggestions.

**Review by 317brian — APPROVED** [review]
LGTM

**Pull request merged**